components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/utils/SubscriptionValidationDataUtil.java

```diff
@@ -132,8 +158,32 @@ public static APIListDTO fromAPIToAPIListDTO(API model) {
                 urlMappingDTO.setThrottlingPolicy(urlMapping.getThrottlingPolicy());
                 urlMappingDTO.setUrlPattern(urlMapping.getUrlPattern());
                 urlMappingDTO.setScopes(urlMapping.getScopes());
+                List<OperationPolicyDTO> operationPolicyDTOList = new ArrayList<>();
+                for(OperationPolicy operationPolicy: urlMapping.getOperationPolicies()) {
+                    OperationPolicyDTO operationPolicyDTO = new OperationPolicyDTO();
+                    operationPolicyDTO.setPolicyId(operationPolicy.getPolicyId());
+                    operationPolicyDTO.setPolicyName(operationPolicy.getPolicyName());
+                    operationPolicyDTO.setPolicyVersion(operationPolicy.getPolicyVersion());
+                    operationPolicyDTO.setDirection(operationPolicy.getDirection());
+                    operationPolicyDTO.setOrder(operationPolicy.getOrder());
+                    operationPolicyDTO.setParameters(operationPolicy.getParameters());
+                    operationPolicyDTOList.add(operationPolicyDTO);
+                }
+                urlMappingDTO.setOperationPolicies(operationPolicyDTOList);
                 urlMappingsDTO.add(urlMappingDTO);
             }
+            List<OperationPolicyDTO> apiPolicies = new ArrayList<>();
+            for(OperationPolicy apiPolicy: model.getApiPolicies()) {
```

Comment:
Formatting issue. You can use https://tharika.medium.com/how-to-setup-code-formatting-in-my-ide-86374ec8fbd2 to fix the formatting using a template instead of requiring to manually do it.